src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -36,183 +36,246 @@
-              <div class="form-group row">
-                <label class="cd-col-form-label required"
-                       for="zonegroupName"
-                       i18n>Zone Name</label>
-                <div class="cd-col-form-input">
-                  <input class="form-control"
-                         type="text"
-                         placeholder="Zone name..."
-                         id="zoneName"
-                         name="zoneName"
-                         formControlName="zoneName">
-                  <cd-help-text>
-                    <span i18n>Enter a unique name for the Zone. A Zone represents a distinct data center or geographical location within a Zonegroup.</span>
-                  </cd-help-text>
-                  <span class="invalid-feedback"
-                        *ngIf="multisiteSetupForm.showError('zoneName', formDir, 'required')"
-                        i18n>This field is required.</span>
-                  <span class="invalid-feedback"
-                        *ngIf="multisiteSetupForm.showError('zoneName', formDir, 'uniqueName')"
-                        i18n>The chosen zone name is already in use.</span>
+                  <select class="form-select"
+                          id="selectedRealm"
+                          formControlName="selectedRealm"
+                          name="selectedRealm">
+                  <option *ngFor="let realm of realmsInfo"
+                          [value]="realm.realm">
+                        {{ realm.realm }}
+                  </option>
+                  </select>
                 </div>
               </div>
-              <div class="form-group row">
-                <label class="cd-col-form-label required"
-                       for="zone_endpoints"
-                       i18n>Zone Endpoints</label>
-                <div class="cd-col-form-input">
-                  <cd-select-badges id="zone_endpoints"
-                                    [data]="rgwEndpoints.value"
-                                    [options]="rgwEndpoints.options"
-                                    [customBadges]="true">
-                  </cd-select-badges>
-                  <cd-help-text>
-                    <span i18n>Select the endpoints for the Zone. Endpoints are the URLs or IP addresses from which the rgw gateways in that zone can be accessed. You can select multiple endpoints in case you have multiple rgw gateways in a zone</span>
-                  </cd-help-text>
+              <div *ngIf="multisiteSetupForm.get('configType').value === 'newRealm' || !showConfigType">
+                <div class="form-group row">
+                  <label class="cd-col-form-label required"
+                         for="realmName"
+                         i18n>Realm Name</label>
+                  <div class="cd-col-form-input">
+                    <input class="form-control"
+                           type="text"
+                           placeholder="Realm name..."
```

Comment:
```suggestion
```
lets always avoid naming placeholder such that, its literally of no use to user.
The label name in such cases is enough.